docs/index.md

```diff
@@ -59,10 +36,15 @@ communication.
 
 For more information and resources, please visit the [Getting Help project page](https://docs.docker.com/project/get-help/).
 
-## Registry documentation
+## Documentation
+
+Basics:
```

Comment:
In the interest of keeping the documentation volume at bay, we should really aim for a separation between use cases. Casual users need no more than deploy and configure.
Everything else is advanced use, and does not deserve a front row seat.